### ISSUE TITLE
6.72 and mira_blob

### ISSUE THREAD

**Issue by arfgh** [sleirsgoevy/ps4jb2#13]
hello there.

Just wanted to know why the  payloads end with a memory error if we try to load them without the  mira loader and via mira_blob. Yes, i am saying mira_blob and not mira_blob_2. This is for 6.72 because in 7.55 all payloads get loadead fine via mira_blob..

Taking in consideration that both exploits were made by you, for sure you will know the reason of this behavior. Of course my intention is to load payloads via mira_blob avoidind the  use of the mira loader. It's funny because, actually in the firmwares 5.05 7.55 and 9.00, we can achive the direct loading for all payloads, but not for 6.72 unfortunatelly. But why ?

thx in advance

**Comment by sleirsgoevy** [comment]
netcat.c from this version of the exploit reserves very little space (64KB) for the payload. It's possible that the payloads you're trying to load do not fit. If you know some payload that should've fit but doesn't, and can test it for me (I don't have a 6.72 PS4 anymore), let me know.

**Comment by arfgh** [comment]
yes, i know if, for that reason i change that length before to proceed to load the netcat code. The size on all payload loadings is the payload size. That works in 7.55, but not on 6.72, well yes for only 2, goldhen and ps4debug. But not for any of the others. And that's the question, why yes in 7.55 and not in 6.72.

**Comment by sleirsgoevy** [comment]
@arfgh just checked it, the 7.55 version of netcat.c allocates 128K of space instead of 64. 6.72 one should too, probably I forgot to recompile the 6.72 code when I did the change

**Comment by arfgh** [comment]
in the netcat.c of both sources, they seem to be the same... so, if there is some change, it should be in the compiled js ? the operation i perfrom before to load, in both cases, it is to change 131072 by the payload size. In the netcat.js of both that is 5 changes.

**Comment by viggen66** [comment]
sleirsgoevy, 

Just an humble remark, on the link for ps4jb2, when you press "672 @ 16f2e88 - 672: fix offline cache", unfortunately the link is dead.

Thanks

**Comment by arfgh** [comment]
@sleirsgoevy so, any possible solution to equal in this detail to the 7.55 netcat ?

**Comment by arfgh** [comment]
bump ?
